app/helpers/session_helper.rb

```diff
@@ -0,0 +1,17 @@
+module SessionHelper
```

Comment:
I had to move these functions out of the session decorator in order for code climate to pass it. Was getting too many method error. 

Both session decorators need to have the same functions for tests to pass.

Reply:
In this case, I think it's better to ignore the "too many methods" error. A module that is only included in one class is a code smell. If a class truly has too many methods, the solution isn't to move those methods to a module and then include the module back into the class. The proper solution would be to create a new class if the original class truly had too many responsibilities.